tensorflow/contrib/cudnn_rnn/python/kernel_tests/cudnn_rnn_ops_test.py

```diff
@@ -94,7 +95,8 @@ def RunLSTM(sess,
   initial_h_op = variable_scope.get_variable(
       "initial_h_op",
       initializer=np.random.rand(batch_size,
-                                 num_units).astype(dtype.as_numpy_dtype),
+                                 num_proj if num_proj else num_units)
```

Comment:
There are 8 occurrence of pattern `num_proj if num_proj else xxx`, can we canonicalize them at the very beginning instead of repeating them this many times?